### ISSUE TITLE
Multicolor change layout without messing the other parts 

### ISSUE THREAD

**Issue by dunnodun** [copycat-killer/awesome-copycats#150]
So there is this weird interaction in Multicolor: 
If you comment out "awful.layout.suit.floating," and go to the workspace "other" the selected layout will be called "no name" and I can't use "Modkey + Space" to switch between layouts. This gets more annoying because if you switch back to an other workspace the layout will still be in this mode of "no name" and I can't do anything else than "Modkey + R" and have to avoid switching to workspace "other". 
Does anybody know, why this happens? 

> awful.layout.layouts = {
> --    awful.layout.suit.floating,
>     awful.layout.suit.tile,
>     awful.layout.suit.tile.left,
>     awful.layout.suit.tile.bottom,
>     awful.layout.suit.tile.top,
>     awful.layout.suit.fair,
>     -- awful.layout.suit.fair.horizontal,
>     -- awful.layout.suit.spiral,
>     -- awful.layout.suit.spiral.dwindle,
>     -- awful.layout.suit.max,
>     -- awful.layout.suit.max.fullscreen,
>     -- awful.layout.suit.magnifier,
>     -- awful.layout.suit.corner.nw,
>     -- awful.layout.suit.corner.ne,
>     -- awful.layout.suit.corner.sw,
>     -- awful.layout.suit.corner.se,
> }

**Comment by copycat-killer** [comment]
It's simple: the template assigns each layout to a tag. They are n: if you comment one without uncommenting another one, they will be n-1 to be matched to n.

This happens with any config.

The layouts number must match the tags number.

**Comment by dunnodun** [comment]
Sorry for my low experience with the config-file, but I don't find the number you mentioned. Could you please point me to line where the number is declared? 
Thank you for your help.

**Comment by copycat-killer** [comment]
Let me try again. You commented `floating`. Then you have two options: remove a tag from the list, for instance: 

```diff
-local tagnames = { "web", "term", "docs", "media", "files", "other" }
+local tagnames = { "web", "term", "docs", "media", "files" }
```

or uncomment another layout, for instance:

```diff
 awful.layout.layouts = {
 --    awful.layout.suit.floating,
     awful.layout.suit.tile,
     awful.layout.suit.tile.left,
     awful.layout.suit.tile.bottom,
     awful.layout.suit.tile.top,
-    -- awful.layout.suit.fair,
+    awful.layout.suit.fair,
     -- awful.layout.suit.fair.horizontal,
     -- awful.layout.suit.spiral,
     -- awful.layout.suit.spiral.dwindle,
     -- awful.layout.suit.max,
     -- awful.layout.suit.max.fullscreen,
     -- awful.layout.suit.magnifier,
     -- awful.layout.suit.corner.nw,
     -- awful.layout.suit.corner.ne,
     -- awful.layout.suit.corner.sw,
     -- awful.layout.suit.corner.se,
 }
```
`tagnames` must have the same number of elements of `awful.layout.layouts`.

**Comment by dunnodun** [comment]
Thank you very much for the clarification :)
